components/style/properties/longhand/ui.mako.rs

```diff
@@ -83,4 +83,21 @@ ${helpers.single_keyword("-moz-window-shadow", "none default menu tooltip sheet"
             _ => Err(()),
         }
     }
+
+    #[cfg(feature = "gecko")]
```

Comment:
@nox thank you for your comment!
Ah, it looks we can drop.
I will re-push after fix this.